build/docker/zap-baseline.py

```diff
@@ -86,6 +90,8 @@ def usage():
     print ('    -i                default rules not in the config file to INFO')
     print ('    -j                use the Ajax spider in addition to the traditional one')
     print ('    -l level          minimum level to show: PASS, IGNORE, INFO, WARN or FAIL, use with -s to hide example URLs')
+    print ('    -n context_file   context file which will be loaded prior to spidering the target')
+    print ('    -p progress_file  progress file which specifies issues that are being addressed')
```

Comment:
Are you going to add an example file? Or some details here about the format of the file? Reading further it seems like json of some sort but that doesn't really provide enough detail.